src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      action="http://localhost:8080/"
+      method="post"
+      class="form first_form"
+    >
+      <div class="input__wrapper__big">
+        <img src="./images/Search.svg" alt="Icon search">
+        <input
+          class="large large_icon"
+          type="search"
+          name="large"
+          id="large"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </div>
+    </form>
+
+    <form
+      action="http://localhost:8080/"
+      method="post"
+      data-qa="small"
+      class="form"
+    >
+      <div class="input__wrapper__small">
+        <img src="./images/Search.svg" alt="Search icon">
+        <input
+        class="small small_icon"
+        type="search"
+        name="small"
+        id="small"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
```

Comment:
```suggestion
        <input
          class="small small_icon"
          type="search"
          name="small"
          id="small"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
pay attention to the code style, fix it all over the page